pkg/services/check/check_suite_test.go

```diff
@@ -0,0 +1,13 @@
+package check
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+func TestCheck(t *testing.T) {
+	RegisterFailHandler(Fail)
+	RunSpecs(t, "Flux Suite")
```

Comment:
Should this be Check Suite instead of Flux Suite?

Reply:
yes